.gitignore

```diff
@@ -5,6 +5,7 @@ package-lock.json
 *.tern-port
 */**/.tern-port
 .DS_Store
+.idea
```

Comment:
This one is not necessary.

Reply:
Oh. Could you explain why?

Reply:
Oh, you use Golang IDE, right?

Reply:
Yeah, git keeps showing .idea file for me. So I added this 😂